engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowLifecycleActor.scala

```diff
@@ -65,6 +66,12 @@ trait WorkflowLifecycleActor[S <: WorkflowLifecycleActorState] extends LoggingFS
   def successResponse: WorkflowLifecycleSuccessResponse
   def failureResponse(reasons: Seq[Throwable]): WorkflowLifecycleFailureResponse
 
+  override def supervisorStrategy = OneForOneStrategy() {
+    case ex: ActorInitializationException =>
+      context.parent ! failureResponse(Seq(ex))
+      Stop
+    case t => super.supervisorStrategy.decider.applyOrElse(t, (_: Any) => Escalate)
```

Comment:
What does this get escalated to? I can't see a supervisor strategy in WorkflowActor?